client-spark/common/src/main/java/org/apache/spark/shuffle/writer/WriteBufferManager.java

```diff
@@ -409,9 +409,15 @@ public List<AddBlockEvent> buildBlockEvents(List<ShuffleBlockInfo> shuffleBlockI
                 + " bytes");
         // Use final temporary variables for closures
         final long _memoryUsed = memoryUsed;
+        final List<ShuffleBlockInfo> _blocks = shuffleBlockInfosPerEvent;
```

Comment:
Could you give a name to keep consistent naming style with other variables?

Reply:
just temporary var.

Reply:
In our project, we never use `_xxxx` as temporary var.